onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -1807,11 +1807,20 @@ ORT_API_STATUS_IMPL(OrtApis::SessionGetProfilingStartTimeNs, _In_ const OrtSessi
 // End support for non-tensor types
 
 #ifndef USE_CUDA
-ORT_API_STATUS_IMPL(OrtApis::OrtSessionOptionsAppendExecutionProvider_CUDA,
-                    _In_ OrtSessionOptions* options, _In_ OrtCUDAProviderOptions* cuda_options){
-    ORT_UNUSED_PARAMETER(options);
-    ORT_UNUSED_PARAMETER(cuda_options);
-    return CreateStatus(ORT_FAIL, "CUDA execution provider is not enabled.");
+ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider_CUDA,
+                    _In_ OrtSessionOptions* options, _In_ const OrtCUDAProviderOptions* cuda_options) {
+  ORT_UNUSED_PARAMETER(options);
+  ORT_UNUSED_PARAMETER(cuda_options);
+  return CreateStatus(ORT_FAIL, "CUDA execution provider is not enabled.");
+}
+#endif
+
+#if defined(ORT_MINIMAL_BUILD)
+ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider_OpenVINO,
```

Comment:
With the shared provider change, the API to create OpenVINO always exists except in the minimal build (as shared providers don't exist in the minimal build). So this is just so that the API still compiles in the minimal build case.